contrib/init/bitcoind.service

```diff
@@ -36,7 +35,6 @@ TimeoutStopSec=600
 
 # Run as bitcoin:bitcoin
 User=bitcoin
-Group=bitcoin
```

Comment:
This group is useful to access the `bitcoin` user's stuff via group permissions.

Reply:
That reads like you assume I've removed the group, which is not entirely true. Systemd will run the process under the user's default group if no group is explicitly specified. And the default group will most likely be the right group (and it definitely should be).